### PR TITLE
Add 'screen' package to the controller machine

### PR DESCRIPTION
**Pull request by cbbayburt**

### CODE REVIEW

File: README_ADVANCED.md [hunk 1]
```diff
@@ -200,6 +200,16 @@ Get HTML results with:
 scp head-ctl.tf.local://root/spacewalk-testsuite-base/output.html .
 ```
 
+To keep the testsuite running after ending the ssh session using `screen` tool:
+```
+ssh -t head-ctl.tf.local screen run-testsuite
+```
+
+You can detach from the session at anytime using the key sequence `^A d`. To re-attach to the existing session:
```
Comment (MalloZup) [inline README_ADVANCED.md]: is Ctrl + A better?

Comment (cbbayburt) [reply README_ADVANCED.md]: That wouldn't work for Mac keyboards ;)

Comment (MalloZup) [reply README_ADVANCED.md]: ehe

**Comment by moio** [comment]
Thanks!

**Pull request merged**